GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationStyle.swift

```diff
@@ -0,0 +1,103 @@
+        public var titleStyle: TitleStyle
+
+        /// Style for subtitle shown in the confirmation area.
+        public var subtitleStyle: SubtitleStyle
+
+        /// Style for button to check messages.
+        public var checkMessagesButtonStyle: CheckMessagesButtonStyle
+
+        /// View's background color
+        public var backgroundColor: UIColor
+    }
+}
+
+extension SecureConversations.ConfirmationStyle {
+    /// Style for title shown in the confirmation area.
+    public struct TitleStyle: Equatable {
+        /// Title text value.
+        public var text: String
+        /// Title font.
+        public var font: UIFont
+        /// Title color.
+        public var color: UIColor
+
+        /// Accessibility properties for SendButtonStyle.
+        public struct Accessibility: Equatable {
+            /// Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+            public var isFontScalingEnabled: Bool
+
+            /// - Parameter isFontScalingEnabled: Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+            public init(isFontScalingEnabled: Bool) {
+                self.isFontScalingEnabled = isFontScalingEnabled
+            }
+
+            /// Accessibility is not supported intentionally.
+            public static let unsupported = Self(isFontScalingEnabled: false)
+        }
+    }
+
+    /// Style for subtitle shown in the confirmation area.
+    public struct SubtitleStyle: Equatable {
+        /// Title text value.
+        public var text: String
+        /// Title font.
+        public var font: UIFont
+        /// Title color.
+        public var color: UIColor
+
+        /// Accessibility properties for SendButtonStyle.
+        public struct Accessibility: Equatable {
+            /// Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+            public var isFontScalingEnabled: Bool
+
+            /// - Parameter isFontScalingEnabled: Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+            public init(isFontScalingEnabled: Bool) {
+                self.isFontScalingEnabled = isFontScalingEnabled
+            }
+
+            /// Accessibility is not supported intentionally.
+            public static let unsupported = Self(isFontScalingEnabled: false)
+        }
+    }
+
+    /// Style for button to check messages.
+    public struct CheckMessagesButtonStyle: Equatable {
+        /// Title text of the button.
+        public var title: String
+        /// Font of the title.
+        public var font: UIFont
+        /// Color of the button title text.
+        public var textColor: UIColor
+        /// Background color of the button.
+        public var backgroundColor: UIColor
```

Comment:
Seems like we need to add `public init`

Reply:
Fixed 3a41ab49d504939b62009e00ff407e7f41f4261a